src/server/auth/cmds/cmds.go

```diff
@@ -658,6 +658,77 @@ func GetRepoRoleBindingCmd(pachCtx *config.Context) *cobra.Command {
 	return cmdutil.CreateAliases(get, "auth get repo", "repos")
 }
 
+// CheckProjectCmd returns a cobra command that sends a GetPermissions request to
+// pachd to determine what permissions a user has on the project.
+func CheckProjectCmd() *cobra.Command {
+	check := &cobra.Command{
+		Use:   "{{alias}} <project> [user]",
+		Short: "Check the permissions a user has on 'project'",
+		Long:  "Check the permissions a user has on 'project'",
+		Run: cmdutil.RunBoundedArgs(1, 2, func(args []string) error {
+			project := client.NewProject(args[0]).AuthResource()
+			c, err := client.NewOnUserMachine("user")
+			if err != nil {
+				return errors.Wrapf(err, "could not connect")
+			}
+			defer c.Close()
+
+			var perms *auth.GetPermissionsResponse
+			if len(args) == 2 {
+				perms, err = c.GetPermissionsForPrincipal(c.Ctx(), &auth.GetPermissionsForPrincipalRequest{
+					Resource:  project,
+					Principal: args[1],
+				})
+			} else {
+				perms, err = c.GetPermissions(c.Ctx(), &auth.GetPermissionsRequest{
+					Resource: project,
+				})
+			}
+			if err != nil {
+				return grpcutil.ScrubGRPC(err)
+			}
+			fmt.Printf("Roles: %v\nPermissions: %v\n", perms.Roles, perms.Permissions)
+			return nil
+		}),
+	}
+	return cmdutil.CreateAliases(check, "auth check project")
+}
+
+// SetProjectRoleBindingCmd returns a cobra command that sets the roles for a user on a project
+func SetProjectRoleBindingCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "{{alias}} <project> [role1,role2 | none ] <subject>",
+		Short: "Set the roles that 'subject' has on 'project'",
+		Long:  "Set the roles that 'subject' has on 'project'",
+		Run: cmdutil.RunFixedArgs(3, func(args []string) error {
+			c, err := client.NewOnUserMachine("user")
+			if err != nil {
+				return errors.Wrapf(err, "could not connect")
+			}
+			defer c.Close()
+
+			var (
+				project *auth.Resource
+				user    string
+				roles   []string
+			)
+			project = &auth.Resource{Type: auth.ResourceType_PROJECT, Name: args[0]}
+			user = args[2]
+			if args[1] != "none" {
```

Comment:
I copied the over from `auth set repo`, but this is my own flavour of it.
Basically instead of
```go
if args[1] == "none" {
    roles = []string{}
} else {
    roles = strings.Split(args[1], ",")
}
```
I have
```go
if args[1] != "none" {
    roles = strings.Split(args[1], ",")
}
```